src/packet.mjs

```diff
@@ -1,9 +1,10 @@
 //shared packet parsing / serialization code
+import {Extension} from "./extensions.mjs"
 
 const text_encoder = new TextEncoder();
-const encode_text = text_encoder.encode.bind(text_encoder);
+export const encode_text = text_encoder.encode.bind(text_encoder);
```

Comment:
I don't really think this needs to be exported when it's used a single time in extensions.mjs.

Reply:
its used multiple times